assets/component-slider.css

```diff
@@ -226,6 +262,23 @@ slider-component.slider-component-full-width {
   .slider.slider--desktop.grid--collapsed .slider__slide:first-child {
     --collapsed-shadow-offset-x: 0px;
   }
+
+  .slider.slider--desktop.grid.grid--collapsed-x .slider__slide.grid__item {
+    --top-left-radius: 0px;
+    --top-right-radius: 0px;
+    --bottom-right-radius: 0px;
+    --bottom-left-radius: 0px;
+  }
+
+  .slider.slider--desktop.grid.grid--collapsed-x .slider__slide.grid__item:first-child {
+    --top-left-radius: var(--border-radius);
+    --bottom-left-radius: var(--border-radius);
+  }
+
+  .slider.slider--desktop.grid.grid.grid--collapsed-x .slider__slide.grid__item:last-child {
+    --top-right-radius: var(--border-radius);
+    --bottom-right-radius: var(--border-radius);
+  }
```

Comment:
Ill create an issue I think where we can look if including the border width into the calculation of the grid item width: 
![](https://screenshot.click/22-56-hw1ra-0owht.png)